DPGAnalysis/MuonTools/python/muNtupleProducerBkg_cff.py

```diff
@@ -11,8 +11,9 @@
 
 def muDPGNanoBkgCustomize(process) :
 
-     if hasattr(process, "NANOAODoutput"):
-          process.NANOAODoutput.outputCommands.append("keep nanoaodFlatTable_*Table*_*_*")
-          process.NANOAODoutput.outputCommands.append("drop edmTriggerResults_*_*_*")
+     for output in ["NANOEDMAODoutput", "NANOAODoutput", "NANOAODSIMoutput"]:
```

Comment:
@battibass Should we add `NANOEDMAODSIMoutput` too?

Reply:
@hqucms I believe it is not strictly needed (the setup is actually not meant to be used in simulation), but as I added `NANOAODSIMoutput`, let me add `NANOEDMAODSIMoutput` as well for completeness.
It might save us from making a further PR in the future.

Reply:
> It might save us from making a further PR in the future.

Yes that's what I was thinking.

Reply:
Should be OK now

Reply:
thanks a lot!